src/test/java/org/jitsi/meet/test/base/AbstractParticipantHelper.java

```diff
@@ -0,0 +1,177 @@
+/*
+ * Copyright @ 2015 Atlassian Pty Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jitsi.meet.test.base;
+
+import org.jitsi.meet.test.util.*;
+import org.openqa.selenium.*;
+
+public abstract class AbstractParticipantHelper
+{
+    /**
+     * The current room name used.
+     */
+    private String currentRoomName;
+
+    /**
+     * The conference first participant(owner) in the tests.
+     */
+    protected Participant<WebDriver> participant1;
+
+    /**
+     * The second participant.
+     */
+    protected Participant<WebDriver> participant2;
+
+    /**
+     * The third participant.
+     */
+    protected Participant<WebDriver> participant3;
+
+    /**
+     * Starts the owner and the seconds participant, if they are not started,
+     * and stops the third participant, if it is not stopped.
+     * participants(owner and 'second participant').
+     */
+    public void ensureTwoParticipants()
+    {
+        waitForParticipant1ToJoinMUC();
+        waitForSecondParticipantToConnect();
+
+        // TODO do we need this???
+        //closeThirdParticipant();
+    }
+
+    /**
+     * Waits until the owner joins the room, creating and starting the owner
+     * if it hasn't been started.
+     */
+    public void waitForParticipant1ToJoinMUC()
+    {
+        if (participant1 == null)
+        {
+            participant1
```

Comment:
Ideally there would be a method "get1stParticipant()" which would check if it's null and create it if needed. I bet there could be more places that need similar logic.